lms/static/sass/discussion/utilities/_variables-v1.scss

```diff
@@ -15,7 +15,7 @@ $forum-color-community-ta: $green-d1 !default;
 $forum-color-marked-answer: $green-d1 !default;
 $forum-color-border: $gray-l3 !default;
 $forum-color-error: $red !default;
-$forum-color-hover-thread: $gray-d3 !default;
+$forum-color-hover-thread: #f6f6f6 !default;
```

Comment:
This is in the non-Pattern Library version, so there probably is a variable I could have used. It isn't affected by the set of grays in the Pattern Library.